### ISSUE TITLE
Players can open chests while being on them

### ISSUE THREAD

**Issue by jasonwynn10** [pmmp/PocketMine-MP#1537]
### Issue description

Players can open chests while being on top of the block. This is not vanilla behavior.

- Expected result: The chest is not able to be opened.
- Actual result: The chest opened and showed its inventory to the client

### Steps to reproduce the issue

1. Stand on chest looking down
2. Tap on chest

### OS and versions

* PocketMine-MP:  This server is running PocketMine-MP 1.7dev-383 「[REDACTED]」 implementing API version 3.0.0-ALPHA9 for Minecraft: PE v1.2.0.81 (protocol version 137)
* PHP: 7.2.0RC4
* Server OS: Windows 10
* Game version: PE & Win10

### Plugins
- [x] Test on a clean server without plugins: is the issue reproducible without any plugins loaded?

If the issue is **not** reproducible without plugins:
- Have you asked for help on our forums before creating an issue?
- Can you provide sample, *minimal* reproducing code for the issue? If so, paste it in the bottom section
- Paste your list of plugins here (use the 'plugins' command in PocketMine-MP)

### Crashdump, backtrace or other files
- Do not paste crashdumps into an issue - please use our Crash Archive at https://crash.pmmp.io for submitting crash reports to not spam the issue tracker. Add links to your reports in the Crash Archive here.
- Please use gist or anything else to add other files and add links here 

* ...

**Comment by jasonwynn10** [comment]
Vanilla behavior prevents a chest from being opened when any entity is on top of it

**Comment by AlexCreeper25** [comment]
Yes, but it's not game-breaking, there lots of more issues that are game-breaking so i think they'll be focusing on them first... This may not be changed soon or may not be changed at all because it's a little insignificant. Btw, it's still good to report issue to make pmmp better.

**Comment by Muqsit** [comment]
Minor correction:
any _living_ entity.
